docs/Getting Started/Installation/index.md

```diff
@@ -1,6 +1,10 @@
 # Installation
 In order to build your app on a SmartDeviceLink (SDL) Core, the SDL software development kit (SDK) must be installed in your app. The following steps will guide you through adding the SDL SDK to your workspace and configuring the environment.
 
+!!! NOTE
+@![iOS]The SDL SDK is currently supported on `iOS 8.0+`.!@@![android]The SDL SDK is currently supported on `Android 2.2 (Froyo)` and above.!@@![javaSE, javaEE]The SDL SDK is currently supported on systems featuring `Java 7 (1.7)` and above.!@
```

Comment:
You can reuse most of the text between the iOS, android and java statements like so:

The SDL SDK is currently supported on @![iOS]iOS 8.0!@@![android]Android 2.2 (Froyo)!@@![javaSE, javaEE]systems featuring Java 7 (1.7)!@ and above.

Reply:
Fixed repeated phrases.